FogCreek/Glitch-Community
# tourmaline-jade: Add <Video> component

## Links
* [tourmaline-jade.glitch.me](https://tourmaline-jade.glitch.me)
* [#157: 
Only load /create videos if they're used for the current viewport width](https://github.com/FogCreek/Glitch-Community-Work/issues/157)

## GIF/Screenshots:
**Multiple video tags, but only one with a source**
<img width="450" src="https://user-images.githubusercontent.com/7584833/61139365-32c06f80-a497-11e9-85d1-1901da3e1452.png" />

## Changes:
This was motivated by improving performance /create, which uses different videos on different viewports, so we only want to load the videos we actually need on a given viewport.

- Add a `<Video>` component that mimics srcset behavior:
```jsx
<Video
  autoPlay
  controls
  track="captions.vtt"
  sources={[
    { src: 'myvideo.mp4', minWidth: 0, maxWidth: 599 }, 
    { src: 'mybigvideo.mp4', minWidth: 600 },
  ]}
/>
```
- Update non-Wistia videos on /create to use the component
- Update CDN URLs to use `CDN_URL` on /create

## How To Test:
Visit [/create](https://tourmaline-jade.glitch.me/create)
- the videos should load
- you should only see the `-small.mp4` videos on small viewports and the big ones on big viewports
- the videos should update when you resize the window
